javascript/src/controllers/hovercard-controller.js

```diff
@@ -26,12 +26,39 @@ export class HovercardController extends Controller {
   }
 
   insertHoverCard (html) {
+    console.log('insertHoverCard')
     const node = stringToDOMNode(html).querySelector('.hovercard')
+    const anode = document.getElementsByTagName('a')[0]
+    const anode_width = anode.offsetWidth
```

Comment:
si tienes duda sobre las convenciones de código en Javascript estamos usando StandardJS y aqui puedes ver todas las reglas: https://standardjs.com/rules.html

También puedes correr `standard --fix` para que te corrija todo lo que puede corregir de forma automática

```suggestion
    const aNodeWidth = anode.offsetWidth
```